src/multibody/fcl.hpp

```diff
@@ -189,23 +152,27 @@ struct GeometryObject
   /// \brief Name of the geometry object
   std::string name;
 
+  /// \brief Index of the parent frame
+  FrameIndex parentFrame;
```

Comment:
This is quite harmless to add this attribute. I dont like the naming. joint and frame parents seems to have the same level of importance, while parentFrame is only a "decoration", not used in any algorithm. I would suggest to keep the naming parent for the joint, and find something more explicit for remembering the body (bodyFrameId?)
parentFrame should be of type BODY, isnt it? If so, it should be explicit in the doc.